pkg/cmd/taskrun/describe.go

```diff
@@ -84,6 +84,10 @@ or
 					if err != nil {
 						return err
 					}
+					if len(trs) == 0 {
+						err = fmt.Errorf("No TaskRuns are present in '%s' namespace", opts.Params.Namespace())
+						return err
```

Comment:
Same comment as for PipelineRuns:

I would just write this to stdout instead of returning an error since this isn't really an error. 

Also suggest:

```suggestion
						err = fmt.Errorf("No TaskRuns present in namespace %s", opts.Params.Namespace())
```

Reply:
done

Reply:
Please print the message to stdout instead of returning an error here.

Reply:
Ohh, okay .. done that.